src/nix/shell.md

```diff
@@ -23,6 +23,12 @@ R""(
   Hi everybody!
   ```
 
+* Run multiple commands in a shell environment:
+
+  ```console
+  # nix shell nixpkgs#gnumake -c /bin/sh -c "cd src && make"
```

Comment:
```suggestion
  # nix shell nixpkgs#gnumake -c sh -c "cd src && make"
```

Since this is Nix it would be nice to not depend on `/bin/sh` specifically.